src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -1613,6 +1676,9 @@ private void ImportRethrow()
 
         private void ImportSizeOf(int token)
         {
+            TypeDesc type = (TypeDesc)_methodIL.GetObject(token);
+            int size = type.GetElementSize().AsInt;
+            PushExpression(StackValueKind.Int32, "sizeof", LLVM.ConstInt(LLVM.Int32Type(), (ulong)size, LLVMMisc.False), _compilation.TypeSystemContext.GetWellKnownType(WellKnownType.Int32));
```

Comment:
Just call the GetWellKnownType helper method declared in this type?

Reply:
Done